src/library/autodj/autodjprocessor.cpp

```diff
@@ -617,7 +617,7 @@ void AutoDJProcessor::playerPositionChanged(DeckAttributes* pAttributes,
     }
 
     if (m_eState == ADJ_IDLE) {
-        if (!thisDeckPlaying && thisPlayPosition < 1) {
+        if (!thisDeckPlaying) {
```

Comment:
Why did you revert this? Doesn't this break Full Track mode with a transition time of 0?

Reply:
Oh I think this makes sense with the other changes... I have to think about this whole set of changes in more detail...

Reply:
I tested and Full Track mode with a transition time of 0 is working with this PR.